Makefile

```diff
@@ -0,0 +1,48 @@
+VERSION=$(shell git describe --abbrev=0 --tags)
+BUILD=$(shell git rev-parse HEAD)
+DIRBASE=./build
+DIR=${DIRBASE}/${VERSION}/${BUILD}/bin
+
+LDFLAGS=-ldflags "-s -w ${XBUILD} -buildid=${BUILD} -X github.com/jpillora/chisel/share.BuildVersion=${VERSION}"
+GCFLAGS=-gcflags=all=-trimpath=$(GOPATH)
```

Comment:
i dont think this needs to be a GCFLAG anymore, `-trimpath` is now built into `go build`

Reply:
wow - that's awesome to know - updated!